app/assets/javascripts/blots/widgetBlock.js

```diff
@@ -338,7 +338,7 @@ var WidgetBlot = function (_Embed) {
     key: 'formats',
     value: function formats(node) {
       var res = {
-        id: +node.dataset.id
```

Comment:
this causes a **NaN** exception causing this issue:
https://www.pivotaltracker.com/story/show/158020785

Reply:
the IDS are now strings, not integers.